packages/graphiql-react/src/custom.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module '*.svg' {
```

Comment:
Not sure how the build was passing previously without this, but this provides typings for imported SCG files so typescript doesn't complain (even though typescript doesn't compile this, we use `tsc --emitDeclarationOnly` so which fails since typescript doesn't understand how to type svg files)

Reply:
This worked before because we load the `vite/client` types. After reverting https://github.com/graphql/graphiql/pull/2620#discussion_r957142262 this should no longer be necessary